### PR TITLE
Rename `MessageHandler` to `ChannelBase`

### PR DESCRIPTION
**Pull request by SmittieC**
This tackles this (#22 ) refactor. It's a quick win. I think "Channels" makes more sense than "MessageHandler"

### CODE REVIEW

**Review by snopoke — APPROVED** [review]

**Comment by SmittieC** [comment]
@snopoke I'll rather wait for the multi-tenant PR to be merged before I merge this. It will be easier for me to handle merge conflicts than for that branch I think

**Comment by SmittieC** [comment]
rebased. Could probably just have merged

**Pull request merged**
